### PR TITLE
upgrade feeluown

### PR DESCRIPTION
**Pull request by iosmanthus**

### CODE REVIEW

**Comment by iosmanthus** [comment]
@berberman seems the ci script needs to be updated. I am running `nixos-unstable` but I can pass the build.

**Comment by berberman** [comment]
Thanks for the PR; you'll need to run `nix flake update` to update nixpkgs pinned in this flake

File: Config.hs [hunk 1]
```diff
@@ -20,8 +20,7 @@ nixSources = do
           package ("feeluown-" <> x)
             `fromPypi` ("fuo_" <> unPkgName x)
 
-  -- https://github.com/NixOS/nixpkgs/issues/119284
-  def $ package "feeluown-core" `sourceManual` "3.7.5" `fetchPypi` "feeluown"
+  def $ package "feeluown-core" `sourceManual` "3.7.6" `fetchPypi` "feeluown"
```
Comment (berberman) [inline Config.hs]: Please uncomment the next line to make it track the latest version, instead of specifying 3.7.6 manually

Comment (iosmanthus) [reply Config.hs]: @berberman PTAL

**Pull request merged**